dev_guide/routes.adoc

```diff
@@ -133,6 +133,18 @@ TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
 
+[[creating-routes-shards]]
+
+== Creating Routes with router shards
```

Comment:
- The conceptual construct of "router sharding" involves labels on routes, it's true.  However, the mere existence of labels on a route does not constitute a router shard, only the *possibility* (given a corresponding selection expression in the router object) of forming a router shard.  How about changing the header to "Creating Routes with Labels" and then explaining this in the paragraph (or better yet, linking to the "Router Sharding" section in Architecture -> Core Concepts -> Routes?

- No need for blank line between the anchor and the header.